### ISSUE TITLE
Confusing information about TF 1118 and user databases

### ISSUE THREAD

**Issue by kalendelaney** [MicrosoftDocs/sql-docs#5597]
You have this paragraph:

> Starting with SQL Server 2016 (13.x), the functionality provided by TF 1118 is automatically enabled for tempdb. For user databases, this behavior is controlled by the SET MIXED_PAGE_ALLOCATION option of ALTER DATABASE, with the default value set to OFF, and trace flag 1118 has no effect. For more information, see ALTER DATABASE SET Options (Transact-SQL).

The first sentence makes it sound like the functionality provided by TF 1118 is only available in tempdb. However,  although user databases ALSO have the same functionality provided by default but it not controlled by the TF, it is controlled by the ALTER DATABASE statement as indicated.  But 'the functionality' (by default) is THE SAME for tempdb and all user databases.  

I would reword to this:

> Starting with SQL Server 2016 (13.x), the functionality provided by TF 1118 is automatically enabled for tempdb and all user databases. For user databases, this behavior is controlled by the SET MIXED_PAGE_ALLOCATION option of ALTER DATABASE, with the default value set to OFF, and trace flag 1118 has no effect. For more information, see ALTER DATABASE SET Options (Transact-SQL).

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0b2881d6-51e3-3eee-a6df-911463517b54
* Version Independent ID: 9295888e-8884-7d3f-7bd4-2bdeccae6962
* Content: [Pages and Extents Architecture Guide - SQL Server](https://docs.microsoft.com/en-us/sql/relational-databases/pages-and-extents-architecture-guide?view=sql-server-ver15)
* Content Source: [docs/relational-databases/pages-and-extents-architecture-guide.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/relational-databases/pages-and-extents-architecture-guide.md)
* Product: **sql**
* Technology: ****
* GitHub Login: @rothja
* Microsoft Alias: **jroth**

**Comment by WilliamAntonRohm** [comment]
@kalendelaney -- Kalen, thank you for your feedback.  I am updating this article.

**Comment by WilliamAntonRohm** [comment]
@rothja -- Jason, please review [private PR 17653](https://github.com/MicrosoftDocs/sql-docs-pr/pull/17653).

**Comment by WilliamAntonRohm** [comment]
Thank you for your feedback.  I have updated the documentation and it should be live by tomorrow.
